tests/DCPS/XTypes/run_test.pl

```diff
@@ -165,7 +165,25 @@
     "NoXTypesNoMatchTypeNames" => {reader_type => "AppendableStructNoXTypes", writer_type => "AppendableStructNoXTypes",
                                    r_reg_type => "ReaderTypeName", w_reg_type => "WriterTypeName",
                                    expect_to_fail => 1, topic => "NoXTypesNoMatchTypeNames_nn", key_val => 1,
-                                   r_ini => "rtps_disc_no_xtypes.ini", w_ini => "rtps_disc_no_xtypes.ini"}
+                                   r_ini => "rtps_disc_no_xtypes.ini", w_ini => "rtps_disc_no_xtypes.ini"},
+    "Tcp_AppendableMatch"         => {reader_type => "AppendableStruct", writer_type => "AdditionalPostfixFieldStruct",
+                                  expect_to_fail => 0, topic => "AppendableStructT", key_val => 4,
+                                  r_ini => "tcp.ini", w_ini => "tcp.ini"},
```

Comment:
I should have said something about this before when the test was first added, but can you make your new entries something like:

```Perl
    "Tcp_AppendableMatch" => {
      reader_type => "AppendableStruct", writer_type => "AdditionalPostfixFieldStruct",
      expect_to_fail => 0, topic => "AppendableStructT", key_val => 4,
      r_ini => "tcp.ini", w_ini => "tcp.ini",
    },
```

That way they're easier to copy, expand, and rename without worrying about indent. You don't have to apply this to the old hash entries if you don't want to.